t/t5319-multi-pack-index.sh

```diff
@@ -165,10 +165,37 @@ test_expect_success 'write midx with twelve packs' '
 
 compare_results_with_midx "twelve packs"
 
+test_expect_success 'write midx with progress off for redirected stderr' '
+	git multi-pack-index --object-dir=$objdir write 2>err &&
+	test_line_count = 0 err &&
+	midx_read_expect 12 74 4 $objdir
+'
+
+test_expect_success 'write midx with the --no-progress option' '
+	git multi-pack-index --object-dir=$objdir write  --no-progress 2>err &&
```

Comment:
nit: an extra space between `write` and `--no-progress`.

Reply:
And this would be more interesting if you had the test check the `--progress` option _does_ output something into `err`.

Reply:
> And this would be more interesting if you had the test check the --progress option does output something into err.

I agree.  As discussed in a separate comment I will drop the tests I've added for the actions that don't currently have progress indicators.